plugins/tech-insights/src/components/CheckResultRenderer.tsx

```diff
@@ -26,6 +26,7 @@ import { BooleanCheck } from './BooleanCheck';
 export type CheckResultRenderer = {
   type: string;
   component: (check: CheckResult) => React.ReactElement;
+  description: (check: CheckResult) => string | React.ReactElement;
```

Comment:
Perhaps this should be optional? It's already handled as such.

Reply:
That makes sense. And it would avoid making it a breaking change for that people. Let me change it

Reply:
Done! Tested and still works as expected.

I also updated the changeset, README and api-report files.